hw/top_earlgrey/data/xbar_main.hjson

```diff
@@ -80,17 +80,21 @@
       clock:     "clk_main_i",
       reset:     "rst_main_ni",
       inst_type: "rv_plic",
-      base_addr: "0x40090000",
-      size_byte: "0x1000",
+      addr_range: [{
```

Comment:
nit: this jumps out a bit as it uses a different bracket placing as the other instances... maybe align the two styles...